samcli/commands/check/bottle_necks.py

```diff
@@ -0,0 +1,106 @@
+"""
+Bottle neck questions are asked here. Data is saved in graph, but not calcualted here.
+"""
+import click
+
+from samcli.commands.check.resources.Graph import Graph
+from samcli.commands.check.resources.LambdaFunction import LambdaFunction
+from samcli.commands._utils.resources import AWS_LAMBDA_FUNCTION
+
+
+class BottleNecks:
+    _graph: Graph
+    _lambda_max_duration: int
+
+    def __init__(self, graph: Graph):
+        """
+        Args:
+            graph (Graph): The graph object. This is where all of the data is stored
+        """
+        self._graph = graph
+        self._lambda_max_duration = 900000
+
+    def ask_entry_point_question(self) -> None:
+        """
+        User is asked which entry point they'd like to start with
+        """
+        entry_points = self._graph.entry_points
+
+        # All entry points must be calcualted before info can be displayed
+        while entry_points:
+            entry_point_question = (
+                "We found the following resources in your application that could be the entry point for a request."
+            )
+
+            item_number = 0
+
+            for item_number, item in enumerate(entry_points):
+                item_name = item.resource_name
+                entry_point_question += "\n[%i] %s" % (item_number + 1, item_name)
+                item_number += 1
```

Comment:
Any reason to increment `item_number` at the end of the loop? `enumerate` does it for you already.